<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="w_bg" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
      <item>
       <widget class="QWidget" name="menu_bar" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="2,5">
           <item>
            <widget class="QLabel" name="lab_item_01">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="title_label">
             <property name="styleSheet">
              <string notr="true">font-family: &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>TFTPClient</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QToolButton" name="btnCancel">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/restart.png</normaloff>:/icons/restart.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="btnDownload">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Downlaod</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/start.png</normaloff>:/icons/start.png</iconset>
             </property>
             <property name="autoRepeatDelay">
              <number>308</number>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="btnUpload">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Upload</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/start.png</normaloff>:/icons/start.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="btnClear">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/clear.png</normaloff>icons/clear.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Upload/Download Progress</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>223</width>
             <height>44</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_littleshow">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/minimize.png</normaloff>icons/minimize.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="cpr_label">
           <property name="text">
            <string>copyrightÂ©2022</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="main_bar" native="true">
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0">
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="7,1">
           <item>
            <widget class="QTextEdit" name="textEditServer">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:5px;
padding: 1px 1px 1px 1px;  
background-color:rgb(255,255,255)
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnServer">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Server</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/gear.png</normaloff>:/icons/gear.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1">
           <item>
            <widget class="QTextEdit" name="textEditServerFile">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:5px;
padding: 1px 1px 1px 1px;  
background-color:rgb(255,255,255)
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnServerFile">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Server File</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTreeWidget" name="treeWidget">
           <property name="styleSheet">
            <string notr="true">QTreeWidget{
border: 1px solid grey ;  
border-radius:20px;
padding: 1px 12px 1px 12px;  
background-color:rgb(255,255,255)
}</string>
           </property>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
            <property name="font">
             <font>
              <family>Perpetua</family>
              <pointsize>13</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/name.png</normaloff>icons/name.png</iconset>
            </property>
           </column>
           <column>
            <property name="text">
             <string>State</string>
            </property>
            <property name="font">
             <font>
              <family>Perpetua</family>
              <pointsize>13</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/property.png</normaloff>icons/property.png</iconset>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Info</string>
            </property>
            <property name="font">
             <font>
              <family>Perpetua</family>
              <pointsize>13</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/property.png</normaloff>:/icons/property.png</iconset>
            </property>
           </column>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1">
           <item>
            <layout class="QVBoxLayout" name="label_all">
             <item>
              <widget class="QPushButton" name="pushButton_index">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>10</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:12pt;</string>
               </property>
               <property name="text">
                <string>Extended Options</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkBoxTSize">
               <property name="minimumSize">
                <size>
                 <width>4</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <bold>true</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>â§TSizeâ§</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxTSize">
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkBoxBlockSize">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>â§BlockSizeâ§</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxBlockSize">
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkBoxTimeOut">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>â§TimeOutâ§</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxTimeOut">
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0,0,0,0,0">
           <item>
            <widget class="QLabel" name="labelUploadTag">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>âï¼</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUpload">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUploadUnit">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>KB/s</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDownloadTag">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>âï¼</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDownload">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDownloadUnit">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>KB/s</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMode">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbBoxMode">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
           <item>
            <widget class="QTextEdit" name="textEditClientFile">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:5px;
padding: 1px 1px 1px 1px;  
background-color:rgb(255,255,255)
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClientFile">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Client File</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
